charts/opentelemetry-kube-stack/templates/bridge.yaml

```diff
@@ -0,0 +1,127 @@
+    {{- end }}
+spec:
+  endpoint: {{ required "opamp endpoint required" $.Values.opAMPBridge.endpoint }}
+  {{- with $.Values.opAMPBridge.headers }}
+  headers:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with $.Values.opAMPBridge.capabilities }}
+  capabilities:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with $.Values.opAMPBridge.componentsAllowed }}
+  componentsAllowed:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with $.Values.opAMPBridge.nodeSelector }}
+  nodeSelector:
+  {{- toYaml . | nindent 4}}
+  {{- end }}
+  replicas: 1
+  {{- if $.Values.opAMPBridge.serviceAccount }}
+  serviceAccount: {{ $.Values.opAMPBridge.serviceAccount }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.image.digest }}
+  image: "{{ $.Values.opAMPBridge.image.repository }}@{{ $.Values.opAMPBridge.image.digest }}"
+  {{- else }}
+  image: "{{ $.Values.opAMPBridge.image.repository }}:{{ $.Values.opAMPBridge.image.tag | default $.Chart.AppVersion }}"
+  {{- end }}
+  {{- if $.Values.opAMPBridge.upgradeStrategy }}
+  upgradeStrategy: {{ $.Values.opAMPBridge.upgradeStrategy }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.imagePullPolicy }}
+  imagePullPolicy: {{ $.Values.opAMPBridge.imagePullPolicy }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.hostNetwork }}
+  hostNetwork: {{ $.Values.opAMPBridge.hostNetwork }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.priorityClassName }}
+  priorityClassName: {{ $.Values.opAMPBridge.priorityClassName }}
+  {{- end }}
+  {{- with $.Values.opAMPBridge.securityContext }}
+  securityContext:
+  {{- toYaml . | nindent 4}}
+  {{- end }}
+  {{- with $.Values.opAMPBridge.podAnnotations }}
+  podAnnotations:
+  {{- toYaml . | nindent 4}}
+  {{- end }}
+  {{- with $.Values.opAMPBridge.resources }}
+  resources:
+  {{- toYaml . | nindent 4}}
+  {{- end }}
+ {{- with $.Values.opAMPBridge.affinity }}
+  affinity:
+  {{- toYaml . | nindent 4}}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.tolerations }}
+  tolerations:
+  {{- with $.Values.opAMPBridge.tolerations }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.volumes }}
+  volumes:
+  {{- with $.Values.opAMPBridge.volumes }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.topologySpreadConstraints }}
+  topologySpreadConstraints:
+  {{- with $.Values.opAMPBridge.topologySpreadConstraints }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.volumeMounts }}
+  volumeMounts:
+  {{- with $.Values.opAMPBridge.volumeMounts }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
+  {{- if $.Values.opAMPBridge.ports }}
+  ports:
+  {{- with $.Values.opAMPBridge.ports }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
+  env:
+  {{- include "opentelemetry-kube-stack.renderenvs" (dict "extraEnvs" $.Values.extraEnvs "env" .Values.opAMPBridge.env) | nindent 4 }}
```

Comment:
Lets put the static `valueFrom` env vars first so that users can refernce them in `extraEnvs`.